### ISSUE TITLE
FR: Support ComfoCool Q600

### ISSUE THREAD

**Issue by middelink** [michaelarnauts/home-assistant-comfoconnect#15]
I own a ComfoCool which I to control or at least make its state visible in Home Assistant.

So far I have found out sensor_802 is the condensor temperature and sensor_85 seems to be the ComfoCool Next Change state (it counts down when I turn the comfocool off for say 1 hour).

sensors_53 and sensor_341 seem related, at least the app changes both it seems.

sensor_53:
  -1 > auto
   1 > off

sensor _341:
  auto -> 0
  off for xx -> 2

Attached are a decoded pcap and the pcap file for good measure where I play with first turning the compfocool off for 1 hour, the continiously off then back to auto and I change the temperature profile to "cool", which first turns bypass to 100% as it is chilly right now, but about 1 minute later the comfocool kicks in.

Le tme know if you need any other information to integrate the comfocool!

[comfoconnect.txt](https://github.com/michaelarnauts/home-assistant-comfoconnect/files/11574113/comfoconnect.txt)
[comfoconnect.pcap.zip](https://github.com/michaelarnauts/home-assistant-comfoconnect/files/11574121/comfoconnect.pcap.zip)

**Comment by gertst** [comment]
This would be a very nice addition, indeed! If you need testers, please let me know!

**Comment by michaelarnauts** [comment]
More information here: https://github.com/michaelarnauts/comfoconnect/issues/27

**Comment by szibis** [comment]
This may help here https://github.com/michaelarnauts/home-assistant-comfoconnect/pull/51 and https://github.com/michaelarnauts/home-assistant-comfoconnect/pull/52

**Comment by szibis** [comment]
Still I think thing that I am not fully sure yet how to register comfocool as seperate device but will dig into this somehow.

**Comment by gertst** [comment]
I don't think it should be a separate device. 
In fact, the Zehnder app just shows it as a property of the ComfoCool: In the app, "ComfoCool" is shown under the "Tasks" tab, more or less like the "Boost" task. So there is also a duration feature, but that's not required in HA: A simple switch entity is all we need: more fancy stuff can then be done using HA automations.
At least this is my take on it.
If you need a tester, I happy to help.

**Comment by szibis** [comment]
In Unit status shows as a part of whole as it is reporting the temp and humidity going through comfocool when working. Still also has separate serial number and own firmware whoch indicates as separate device from perspective of how comfoair and comfolan c is reported ![image](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/329831/05dc2062-b283-416c-a136-1ce0ec7c2104)

**Comment by szibis** [comment]
I will start full tests once we will have new release of aiocomfoconnect, but all help is appreciated.

**Comment by michaelarnauts** [comment]
I have merged all your commits and released a new aiocomfoconnect. You can try installing the master branch of this integration in HACS to have these changes in HACS.

@szibis I want to thank you for your invested time in this!

**Comment by michaelarnauts** [comment]
> In Unit status shows as a part of whole as it is reporting the temp and humidity going through comfocool when working. Still also has separate serial number and own firmware which indicates as separate device from perspective of how comfoair and comfolan c is reported ![image](https://private-user-images.githubusercontent.com/329831/330178603-05dc2062-b283-416c-a136-1ce0ec7c2104.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NjYxNjYsIm5iZiI6MTcxNTk2NTg2NiwicGF0aCI6Ii8zMjk4MzEvMzMwMTc4NjAzLTA1ZGMyMDYyLWIyODMtNDE2Yy1hMTM2LTFjZTBlYzdjMjEwNC5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTdUMTcxMTA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGM1YzViZTJhMTFkM2FlMmRlNDY3NmJhM2ZkNDdhZDdiMTEzZjJjMWRiMjUyZDBjZTMwNzc1YmFkNGNkMTcyMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.mvsAXle_pX3yoFX1QQDyp3n5gPlwNvPlTkmCX0XnvTk)

This is probably another Node. See https://github.com/michaelarnauts/aiocomfoconnect/blob/master/docs/PROTOCOL-RMI.md#nodes.

During connection, the bridge sends a CnNodeNotificationType to indicate what nodes are available.

I assume you can get the version numbers from that Node, It's probably on SubUnti 0x01, property 0x06. This is the firmware version for Node 0x01 (The Unit itself) and 0x30 (The Comfoconnect Bridge).

**Comment by szibis** [comment]
Yes will try to investigate this to register this properly as it is in Bridge. For now adding some improvements to have that sensors in ComfoAir, but once I add new device will move the sensors to this new device and also mode selector.

**Comment by szibis** [comment]
Just for now current master, before above PR for small changes reports properly state and condensor temperature 
<img width="564" alt="image" src="https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/329831/64295963-ca1c-4a00-8a8f-dcfe954b700a">

**Comment by szibis** [comment]
Ok and state now reported properly 
<img width="566" alt="image" src="https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/329831/8a70f367-bce9-430c-af9d-5d761b22acb9">
